cluster/src/main/java/org/apache/iotdb/cluster/query/fill/ClusterPreviousFill.java

```diff
@@ -240,19 +242,28 @@ private ByteBuffer remoteAsyncPreviousFill(
   private ByteBuffer remoteSyncPreviousFill(
       Node node, PreviousFillRequest request, PreviousFillArguments arguments) {
     ByteBuffer byteBuffer = null;
-    try (SyncDataClient syncDataClient =
-        metaGroupMember
-            .getClientProvider()
-            .getSyncDataClient(node, RaftServer.getReadOperationTimeoutMS())) {
-
-      byteBuffer = syncDataClient.previousFill(request);
-    } catch (Exception e) {
+    SyncDataClient client = null;
+    try {
+      client =
+          metaGroupMember
+              .getClientProvider()
+              .getSyncDataClient(node, RaftServer.getReadOperationTimeoutMS());
+      byteBuffer = client.previousFill(request);
+    } catch (IOException e) {
+      logger.warn("{}: Cannot connect to {} during previous fill", metaGroupMember, node);
```

Comment:
hi , xinyu , I have a little question.
why does IOException  not deal?  only print warn

Reply:
To maintain compatibility. Before this PR, this function will print a row of log and return null when it fails to get the client